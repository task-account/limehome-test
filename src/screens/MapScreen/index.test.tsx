import renderer from "react-test-renderer";
import { MapScreen } from ".";

jest.mock("@utils/hooks", () => ({
  withStyles: () => () => ({ useStyles: jest.fn() })
}));

describe(`MapScreen`, () => {
  it("render", () => {
    const tree = renderer.create(<MapScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
