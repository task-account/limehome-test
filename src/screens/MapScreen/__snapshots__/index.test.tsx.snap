// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapScreen render 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <AIRMap
    edgePadding={
      Object {
        "bottom": 50,
        "left": 50,
        "right": 50,
        "top": 50,
      }
    }
    initialRegion={
      Object {
        "latitude": 52.520008,
        "latitudeDelta": 0.0922,
        "longitude": 13.404954,
        "longitudeDelta": 0.0421,
      }
    }
    mapRef={[Function]}
    onChange={[Function]}
    onLayout={[Function]}
    onMapReady={[Function]}
    onMarkerPress={[Function]}
    onRegionChangeComplete={[Function]}
    region={null}
  />
</RCTSafeAreaView>
`;
